### PR TITLE
Global PgBouncerCompatibility avoids extra_float_digits conn param

### PR DESCRIPTION
**Pull request by efepapa**
This PR adds a simple, global way to avoid sending `extra_float_digits` param, so we can use pq with PgBouncer's default configuration.

@potocnyj PTAL

### CODE REVIEW

File: conn.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ var (
 	ErrSSLKeyHasWorldPermissions = errors.New("pq: Private key file has group or world access. Permissions should be u=rw (0600) or less.")
 )
 
+var PgBouncerCompatibility = false
```
Comment (potocnyj) [inline conn.go:35]: Is there going to be a separate PR to expose this as a config flag?

**Review by potocnyj — APPROVED** [review]
Other than the one question LGTM

File: conn.go (hunk 1 above)
Comment (efepapa) [reply conn.go:35]: Currently we don't need it-- PgBouncer fails with a custom error.
